core/src/test/scala/unit/kafka/security/authorizer/AclAuthorizerTest.scala

```diff
@@ -82,16 +86,17 @@ class AclAuthorizerTest extends ZooKeeperTestHarness {
   override def setUp(): Unit = {
     super.setUp()
 
+    val authorizers = Seq(aclAuthorizer, aclAuthorizer2, interfaceDefaultAuthorizer.authorizer)
```

Comment:
Can we move testing of `interfaceDefaultAuthorizer.authorizer` into another class? This is `AclAuthorizerTest` and testing of `interfaceDefaultAuthorizer` seems unrelated to this test.

Reply:
Move the interface default test to a new class. Also, created a util class for code sharing.
commit 6c550fd04a0c1912e669bf18d60dee27dd03e53c